classes/section/Repository.php

```diff
@@ -172,7 +172,7 @@ public function isEmpty(int $sectionId, int $contextId): bool
             ->getCollector()
             ->filterByContextIds([$contextId])
             ->filterBySectionIds([$sectionId])
-            ->getCount() > 0;
+            ->getCount() <= 0;
```

Comment:
Could you update this to `=== 0` for clarity? It should always be numeric, and never <0.

Reply:
Done .